README.md

```diff
@@ -133,13 +134,14 @@ Install this project with:
 
 ## 👥 Authors <a name="authors"></a>
 
+- <a href="https://github.com/kimitawanjohi"> **Kimita Wanjohi** </a> 
 > Mention all of the collaborators of this project.
 
 👤 **Author1**
```

Comment:
- [ ] It would be better if you replace your name with the `Author1` to make your readme more professional :blush: